### DISCUSSION TITLE
Release Artifact on Central

### DISCUSSION THREAD

**Discussion by khmarbaise** [artipie/artipie#1205] [Q&A] [answered]
Hi, is there a particular reason why the current release artifact is not available via central repository?

**Comment by ChGen** [comment, marked as answer]
Good point, we will check few things a bit later and publish Artipie artifacts to the central too (#1206).

**Comment by ChGen** [comment]
Artipie v0.28.0 artifacts now available in Central:
https://mvnrepository.com/artifact/com.artipie/artipie
https://repo1.maven.org/maven2/com/artipie/artipie/v0.28.0/
